Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.py

```diff
@@ -1242,6 +1259,56 @@ def get_process_timeline(client, data_args):
     return human_readable, outputs, raw_response
 
 
+def upload_intel_doc(client, data_args):
+    # OpenIOC format is XML
+    headers = {
+        'Content-Type': 'application/xml'
+    }
+    raw_response = client.do_request('POST', '/plugin/products/detect3/api/v1/intels/', data=data_args.get('intel_doc'),
+                                     headers=headers)
+    intel_doc = get_intel_doc_item(raw_response)
+
+    context = createContext(intel_doc, removeNull=True)
+    outputs = {'Tanium.IntelDoc(val.ID && val.ID === obj.ID)': context}
+
+    intel_doc['LabelIds'] = str(intel_doc['LabelIds']).strip('[]')
```

Comment:
What is it used for?

Reply:
I can't remember why I wrote this. It seems to run fine without it so we can remove it.